assets/css/layout/_footer.scss

```diff
@@ -6,90 +6,131 @@ footer {
   margin-top: rem(30);
   padding: rem(16) rem(32);
 
+  .netlify-notice {
+    padding-top: rem(32);
+    text-align: center;
+
+    a {
+      color: $white;
+      font-weight: $font-weight-normal;
+
+      &:hover,
+      &:focus {
+        color: $navy-light;
+      }
+    }
+  }
+
   .footer-content {
     align-items: baseline;
     display: flex;
     flex-direction: column;
     font-size: rem(16);
     line-height: rem(29);
+    margin: auto;
     max-width: $max-width-margins;
 
     h2 {
       font-size: rem(16);
       font-weight: bold;
     }
 
+    .funders {
+      a {
+        color: $white;
+      }
+
+      svg {
+        padding: rem(16) rem(16) rem(16) 0;
+        width: rem(144);
+      }
+    }
+
     .contact-info {
       width: 100%;
 
       a {
+        color: $white;
         font-weight: $font-weight-normal;
-      }
 
-      svg {
-        width: rem(160);
+        svg {
+          padding: rem(32) rem(16) rem(16) 0;
+          width: rem(176);
+        }
       }
 
       .logo-ocadu {
-        padding-top: rem(32);
         text-align: center;
-      }
-    }
 
-    .social-media {
-      align-self: center;
-      padding-bottom: rem(32);
+        a svg {
+          padding: rem(16);
+          width: rem(128);
+        }
+      }
     }
 
     .nav-home-about,
     .nav-tools,
     .nav-our-data,
     .nav-inclusion-challenges {
       display: none;
+    }
 
+    .nav-home-about,
+    .nav-tools,
+    .nav-our-data,
+    .nav-inclusion-challenges,
+    .contact-info,
+    .funders {
       a:hover,
       a:focus {
         color: $navy-light;
+
+        svg: {
+          color: $navy-light;
```

Comment:
Consider using `fill: currentColor` on the SVG: https://css-tricks.com/cascading-svg-fill-color/

Reply:
> Consider using `fill: currentColor` on the SVG: https://css-tricks.com/cascading-svg-fill-color/

Very good point. Following your suggestion of using `currentColor`, I found there's already a similar css set for svg elements: `svg:not([fill]) {fill: currentColor}`. This line of explicitly setting SVG color at hover/focus is unnecessary. I've removed it. Thanks.